openquakeplatform/bin/oq_install.sh

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env bash 
+# GeoNode installer script
+#
+# using getopts
+#
+
+if [ $GEM_SET_DEBUG ]; then
+    set -x
+fi
+# set -e
+
+source ~/env/bin/activate
+
+if [[ $EUID -ne 0 ]]; then
+    echo "This script must be run as root" 1>&2
+    exit 1
+fi
```

Comment:
Many operations you are performing in this script should _not_ be executed as root: first of all you are sourcing an env owned by a user in its home.
You are also performing several `pip install` as root in a venv placed in a user home which should not contain files owned by root.

Reply:
This script is in Geonode package, i have just putted other thinks for the installation